### PR TITLE
git-clone: correct and reword examples.

### PR DESCRIPTION
**Pull request by mebeim**
This page had some wrong and also strangely worded examples. I corrected and reworded them.

### CODE REVIEW

File: pages/common/git-clone.md [hunk 1]
```diff
@@ -11,14 +11,14 @@
...
+`git clone -l {{path/to/local/repository}}`
 
-- Do it quietly:
+- Clone quietly:
 
-`git clone -q`
+`git clone -q {{remote_repository_location}}`
 
-- Clone an existing repository, and truncate to the specified number of revisions, save your time mostly:
+- Clone an existing repository and truncate the history to the 10 most recent commits on the default branch (useful to save time):
```
Comment (agnivade) [inline pages/common/git-clone.md]: This implies that it clones the entire repo, and _then_ truncates the history, which is incorrect. Let us reword to say that it clones the 10 most recent commits of a repository.

Comment (mebeim) [reply pages/common/git-clone.md]: What about this?

    - Clone an existing repository only fetching the 10 most recent [...]

Comment (agnivade) [reply pages/common/git-clone.md]: SGTM

Comment (mebeim) [reply pages/common/git-clone.md]: Done!

**Comment by mebeim** [comment]
Merging as it's been approved 3 times.

**Pull request merged**